posthog/hogql_queries/utils/revenue.py

```diff
@@ -1,95 +0,0 @@
-from typing import Union
```

Comment:
it's a shame that git didn't notice that this was a moved file rather than deleted :(